src/Batch/BatchApiV2.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace Smartling\Batch;
+
+use GuzzleHttp\ClientInterface;
+use JetBrains\PhpStorm\ArrayShape;
+use JetBrains\PhpStorm\ExpectedValues;
+use Psr\Log\LoggerInterface;
+use Smartling\AuthApi\AuthApiInterface;
+use Smartling\BaseApiAbstract;
+use Smartling\Batch\Params\ListBatchesParameters;
+use Smartling\Exceptions\SmartlingApiException;
+
+class BatchApiV2 extends BaseApiAbstract
+{
+    public const ENDPOINT_URL = 'https://api.smartling.com/jobs-batches-api/v2/projects';
+    private const GET_BATCH_STATUS_ARRAY_SHAPE = [
```

Comment:
This is nonsense. Typehinting through the php doc.. It makes no sense for type safety at all. Moreover, works only in php storm. why should I read these lines in another ide?